examples/spring-boot/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,19 @@
+spring:
```

Comment:
this file is not needed, simply put `spring.datasource.driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver` next to `spring.datasource.url` in `AbstractIntegrationTest`